packages/scene-graph-mediator/client/package.json

```diff
@@ -47,14 +47,15 @@
     "sinon": "^6.1.4",
     "ts-loader": "^4.4.2",
     "ts-node": "^8.1.0",
-    "tslint": "^5.11.0",
+    "tslint": "^5.17.0",
     "tslint-config-airbnb": "^5.9.2",
     "typedoc": "^0.11.1",
     "typescript": "^2.9.2",
     "yaml": "^1.5.0"
   },
   "gitHead": "74a6a4cebe353511caddc925538937a265572b5c",
   "dependencies": {
+    "@types/node": "^12.0.4",
```

Comment:
This should be installed as devDependencies. (install with `npm i --save-dev`)
because module for typing is not used in runtime.

Reply:
> should be installed as devDependencies

If needed by `lib/index.d.ts`, I think it will be error when using as npm.

@Egliss 
What kind of error will occur if do not install `@types/node`?

Reply:
@zprodev 
Ah, i didn't understanded the type definition file.
I had thought the module will used by type complete system by code editor.